### PR TITLE
Add LVF - A la Volette Film - Update facilities.json

### PR DESCRIPTION
**Pull request by japierce1028**
Opt IN

### CODE REVIEW

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -3853,6 +3853,16 @@
     "code": "LV",
     "description": "LightVAULT"
   },
+  {
+    "code": "LVF",
+    "contact": {
+      "address": "236 rue de la mairie 73220 Epirre, France",
```
Comment (palemieux) [inline src/main/data/facilities.json:3859]:
```suggestion
      "address": "236 rue de la Mairie, 73220 Epirre, France",
```

**Review by palemieux — CHANGES_REQUESTED** [review]
Is `Epirre` really the town's name? Could it be `Epierre`?

File: src/main/data/facilities.json (hunk 1 above)
Comment (japierce1028) [reply src/main/data/facilities.json:3859]: It's a cut/paste from their entry. Do you want me to ask them?

Comment (palemieux) [reply src/main/data/facilities.json]: > It's a cut/paste from their entry. Do you want me to ask them?

I would definitely ask them to confirm the name of the town.

**Review by palemieux — APPROVED** [review]

**Pull request merged**